logstash/pipeline/logstash.conf

```diff
@@ -10,9 +11,19 @@ input {
 ## Add your filters / logstash plugins configuration here
 
 output {
-	elasticsearch {
-		hosts => "elasticsearch:9200"
-		user => elastic
-		password => "${ELASTIC_USER_PASSWORD:changeme}"
+	if [ENV] {
```

Comment:
Also, if `ENV` does not exist, it's probably because of an application sending logs during e.g. a unit test. Maybe we should drop such messages (where `ENV` is missing).

In all our relevant cases (prod server, CI tests), ENV will be set explicitly. So why allow messages without it?
Hmm, maybe for the case of running the ELK stack locally for debugging a local server/tests...